app/views/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from: ../../.rubocop.yml
+
+# Disable for the `Rubocop` lints in haml-lint
+Style/IfUnlessModifier:
```

Comment:
The rule here is attempting to enforce the "trailing conditional" style where you'd have if/unless after the view output instead of wrapping it. This is (in my opinion) sort of hard to read even when it's just normal ruby code ... and really weird when it's in a view. Seems preferable to disable, which is why I added this here. Open to alternate ideas/opinions.